fix(sdb): add command to check version in QS and how to MTA-3693 (#2164)
scaleway/docs-content: serverless/sql-databases/quickstart.mdx

    </Message>

3. Set up the **Connection string**:

>     - If you already have a secret key, paste it in the connection string in place of the `[INSERT_SECRET_KEY_HERE]` placeholder.

Reviewer: ```suggestion
    - If you already have a secret key, replace the `[INSERT_SECRET_KEY_HERE]` placeholder with your key.
```
